examples/models/file/basic_plot.py

```diff
@@ -1,3 +1,12 @@
+'''A scatter plot of a smooth periodic oscillation. This example demonstrates red circle scatter markers with black outlines, using the low-level bokeh.models API.
+
+.. bokeh-example-metadata::
+    :sampledata: sine wave
+    :apis: bokeh.models
```

Comment:
This could also be some more (specific) APIs:

```suggestion
    :apis: bokeh.models.Circle, bokeh.models.Plot, bokeh.models.ColumnDataSource, bokeh.models.LinearAxis, bokeh.models.PanTool, bokeh.models.WheelZoomTool
```